apiserver/facades/agent/provisioner/provisioninginfo.go

```diff
@@ -57,67 +58,115 @@ func (api *ProvisionerAPI) ProvisioningInfo(args params.Entities) (params.Provis
 	return result, nil
 }
 
-func (api *ProvisionerAPI) getProvisioningInfo(m *state.Machine, env environs.Environ) (*params.ProvisioningInfo, error) {
-	cons, err := m.Constraints()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
+func (api *ProvisionerAPI) getProvisioningInfo(m *state.Machine, env environs.Environ) (*params.ProvisioningInfoV10, error) {
+	var err error
+	result := params.ProvisioningInfoV10{}
```

Comment:
nit: `var result params.ProvisioningInfoV10` (it makes it very obvious that it's not got anything in it just from scan reading)